packages/framework/presence/src/test/mockEphemeralRuntime.ts

```diff
@@ -162,15 +162,16 @@ export class MockEphemeralRuntime implements IEphemeralRuntime {
 
 	public getQuorum: () => ReturnType<IEphemeralRuntime["getQuorum"]>;
 
-	public submitSignal: IEphemeralRuntime["submitSignal"] = (
-		...args: Parameters<IEphemeralRuntime["submitSignal"]>
-	) => {
+	public readonly submittedSignals: Parameters<IEphemeralRuntime["submitSignal"]>[] = [];
+
+	public submitSignal(...args: Parameters<IEphemeralRuntime["submitSignal"]>): void {
+		this.submittedSignals.push(args);
```

Comment:
This syntax doesn't make it readonly?
I used the other syntax to make it clearly replaceable.

Reply:
I can revert these changes. This was from an early draft where I didn't subclass this mock runtime and tried to change it directly. I'll revert - not needed for this PR.